src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -516,7 +516,15 @@ export class ConnectionDialogService implements IConnectionDialogService {
 			}));
 
 		}
-		this._logService.error(message);
-		this._errorMessageService.showDialog(severity, headerTitle, message, messageDetails, actions);
+
+		let errorMessage = message;
+		if (message.includes('could not translate host name')) {
```

Comment:
this doesn't seem reliable to me, we should use error code instead of English version of the error message.